vm/systemSmartContracts/delegation.go

```diff
@@ -1101,11 +1096,19 @@ func (d *delegation) finishDelegateUser(
 }
 
 func (d *delegation) delegate(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
-	if args.CallValue.Cmp(d.minDelegationAmount) < 0 {
-		d.eei.AddReturnMessage("delegate value must be higher than minDelegationAmount " + d.minDelegationAmount.String())
+	delegationManagement, err := d.getDelegationManagement()
+	if err != nil {
+		d.eei.AddReturnMessage("error getting minimum delegation amount " + err.Error())
 		return vmcommon.UserError
 	}
-	err := d.eei.UseGas(d.gasCost.MetaChainSystemSCsCost.DelegationOps)
+
+	minDelegationAmount := delegationManagement.MinDelegationAmount
```

Comment:
No need for this variable

Reply:
added Jira task EN-9067 as to not forget about them